docs/source/api_reference/models.rst

```diff
@@ -266,6 +266,12 @@ WGAN
 .. autoclass:: deepchem.models.WGAN
   :members:
 
```

Comment:
`CNN` should be moved to the torch model section of the doc (https://github.com/deepchem/deepchem/blob/master/docs/source/api_reference/models.rst#pytorch-models) from the existing keras model